Sources/SwiftParser/Declarations.swift

```diff
@@ -465,7 +465,7 @@ extension Parser {
     }
 
     precondition(self.currentToken.starts(with: "<"))
-    let langle = self.consumeAnyToken(remapping: .leftAngle)
+    let langle = self.consumeStartingCharacterOfCurrentToken(as: .leftAngle)
```

Comment:
Can’t you use `consumePrefix` here?

Reply:
Doh! Sorry. Using `consumePrefix` instead.